app/models/booking.rb

```diff
@@ -1,5 +1,8 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :garden
-  has_many :reviews
+  has_many :reviews, dependent: :destroy
```

Comment:
This means that when the booking is destroyed, the reviews associated will be destroyed also.